Sources/PackagePlugin/Triple.swift

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+public struct Triple {
+    public let string: String
```

Comment:
maybe: `description: String` instead? At some point this might want to be strongly typed

Reply:
I think we'd have to replicate the whole `Triple` type we already have in SwiftPM implementation itself, and I'd like to avoid it if possible. If we want to land this in the near term, we'll have to keep it minimal, unblocking plugins to parse the triple as needed on their side.

Reply:
Evolving the `Triple` type, providing a public API for its parsing and other helpers like bit-width should come in future iterations, as Swift Evolution pitches/proposals I'd assume.